.gitignore

```diff
@@ -0,0 +1,3 @@
+/lib
```

Comment:
Are you linting your JS? Using Grunt? If so, and your package.json and Gruntfile are in the lib folder, they aren't being pushed up.